src/modules/app/app.ts

```diff
@@ -185,13 +208,20 @@ export class App {
         window.open(`${window.location.origin}?id=${id}`, '_blank');
     }
 
-    public getSum(): number {
+    public onClickCart(): void {
+        window.location.href = `${window.location.origin}?cart`;
+        this.drawCartPage();
+    }
+
+    public getSum(): number {;
         const array: number[] = this.getSelectedProducts();
         let sum: number = 0;
-        productsData.forEach((item) => {
-            if(array.includes(item.id)) {
-                sum += item.price;
-            }
+        array.forEach((id) => {
+            productsData.forEach((product) => {
```

Comment:
Other thing should catch your eye: 
1. `forEach` inside `forEach` not always, but usually can be done better. 
2. Can we avoid outer `sum` variable?

I think you are trying to get intersection between two sets and get total price.

```ts
const selectedProducts: Set<number> = new Set(this.getSelectedProducts());

return this.getSelectedProducts()
    .filter(product => selectedProducts.has(product))
    .map(product => product.price)
    .reduce((acc, current) => acc + current, 0);
```

Reply:
I couldn't use your recommendation because this way product isn't object.
What do you think about the new implementation?